### PR TITLE
Add missing hover CSS animation to button.tsx

### PR DESCRIPTION
**Pull request by hokwaichan**
# Ticket Link

[1686](https://uhawaii.atlassian.net/browse/GROUPINGS-1686)

# List of squashed commits

-   Add missing hover CSS animation to button.tsx

# Test Checklist

-   [x] Unit Tests Passed
-   [x] Integration Tests Passed
-   [x] General Visual Inspection

### CODE REVIEW

File: ui/src/app/globals.css [hunk 1]
```diff
@@ -11,3 +11,23 @@
...
+    background: -webkit-linear-gradient(to bottom, #7db1c4, #5a9cb4);
+    background: -o-linear-gradient(to bottom, #7db1c4, #5a9cb4);
+    background: -ms-linear-gradient(to bottom, #7db1c4, #5a9cb4);
+    background-repeat: repeat-x;
+    transition: background 0.15s ease-in-out;
+}
+
+.variant-default:hover {
+    background: #004e59;
+}
```
Comment (JorWo) [inline ui/src/app/globals.css:33]: Would it be possible to keep everything else but the `background` and `transition` CSS as Tailwind CSS classes? Then we should rename `.variant-default` to describe this CSS class better. Maybe `.uh-groupings-button-gradient`.

Comment (JorWo) [reply ui/src/app/globals.css:33]: Also it seems that the best practice is to use `@layer components` :mhttps://tailwindcss.com/docs/adding-custom-styles#adding-component-classes

**Review by JorWo — APPROVED** [review]

**Comment by JorWo** [comment]
The gradient seems to be in the wrong direction, I think `background: -moz-linear-gradient(to bottom, #5a9cb4, #7db1c4);` should be correct.

**Comment by JorWo** [comment]
You might want to add `.btn-gradient:focus` so that pressing tab will trigger the animation too:
```
.btn-gradient:hover, .btn-gradient-focus {
    background: #004e59;
}
```

**Comment by JorWo** [comment]
I noticed the "Learn More" button on the home page before login does not have the blue ring for some reason.

**Comment by hokwaichan** [comment]
> I noticed the "Learn More" button on the home page before login does not have the blue ring for some reason.

I think it has to tab twice to show the blue ring

**Review by JorWo — APPROVED** [review]
Looks good!

**Pull request merged**